Sources/Commands/SwiftPackageTool.swift

```diff
@@ -183,7 +183,24 @@ public struct SwiftPackageTool: SwiftTool {
                     return try get(manifest, manifestParser: parseManifest)
                 }
             }
-        
+
+            func findDirectoryReferences(path: String) throws -> [String] {
+                let rootDirectories = walk(path, recursively: false)
+                let rootDirectoriesToConsider = rootDirectories.filter {
+                    if $0.hasSuffix(".xcodeproj") { return false }
+                    if $0.hasSuffix(".playground") { return false }
+                    if $0.basename.hasPrefix(".") { return false }
+                    return $0.isDirectory
```

Comment:
I don't see anything wrong with the list, and I think it's a good start.  It seems to me that there might be a connection here with the initial list of excludes for a `.gitignore` created by `swift package init` and possibly other logic.  I think for this PR it's fine, but in the future we might perhaps want to unify all of these lists or at least correlate them.